finagle-core/src/main/scala/com/twitter/finagle/service/RequeueFilter.scala

```diff
@@ -36,13 +36,14 @@ import com.twitter.util._
  *      for more details.
  */
 private[finagle] class RequeueFilter[Req, Rep](
-    retryBudget: RetryBudget,
-    retryBackoffs: Stream[Duration],
-    statsReceiver: StatsReceiver,
-    canRetry: () => Boolean,
-    maxRetriesPerReq: Double,
-    timer: Timer)
-  extends SimpleFilter[Req, Rep] {
+  retryBudget: RetryBudget,
+  retryBackoffs: Stream[Duration],
+  statsReceiver: StatsReceiver,
+  canRetry: () => Boolean,
+  maxRetriesPerReq: Double,
+  timer: Timer,
+  classifier: ResponseClassifier = PartialFunction.empty
```

Comment:
lets not give a default argument since we do not need to keep backwards compat for package private changes.